hw/ip/rv_dm/dv/cov/cover_reg_top.cfg

```diff
@@ -5,6 +5,12 @@
 +node tb.dut *_tl_*
 +node tb.dut jtag_*
 
+// Non-RTL assertion only blocks.
+-moduletree rv_dm_regs_csr_assert_fpv
+begin assert
+  +moduletree rv_dm_regs_csr_assert_fpv
+end
+
```

Comment:
we may need this one. Only CSR tests can cover those very corner assert coverage for TLUL, like b2b same address accesses
https://github.com/lowRISC/opentitan/blob/63c4132d4cab8cd34afc239c8111f25187816b9a/hw/dv/tools/vcs/cover_reg_top.cfg#L10

Reply:
This cfg file is an addition to `hw/dv/tools/vcs/cover_reg_top.cfg` - it's not a replacement. Please see the change to `rv_dm_sim_cfg.hjson` below. These should get factored in too, right?